cmd/tailscale/cli/file.go

```diff
@@ -263,51 +264,85 @@ func formatIEC(n float64, unit string) string {
 	}
 }
 
-func getTargetStableID(ctx context.Context, ipStr string) (id tailcfg.StableNodeID, isOffline bool, err error) {
+func getTargetStableID(ctx context.Context, ipStr string) (tailcfg.StableNodeID, bool, error) {
```

Comment:
nit: it's nice to have the return parameter names in the function signature. Without them, maybe add a doc comment to explain what the parameters are?